src/python/crash_analysis/stack_parsing/stack_symbolizer.py

```diff
@@ -382,23 +382,25 @@ def symbolize(self, addr, binary, offset):
     """Overrides Symbolizer.symbolize."""
     if self.binary != binary:
       return None
-    if not os.path.exists(binary):
-      # If the binary doesn't exist atos will exit which will lead to IOError
-      # exceptions being raised later on so just don't try to symbolize.
+
+    try:
+      atos_line = self.atos.convert('0x%x' % int(offset, 16))
+      while 'got symbolicator for' in atos_line:
+        atos_line = self.atos.readline()
+      # A well-formed atos response looks like this:
+      #   foo(type1, type2) (in object.name) (filename.cc:80)
+      match = re.match('^(.*) \(in (.*)\) \((.*:\d*)\)$', atos_line)
+      if match:
+        function_name = match.group(1)
+        function_name = re.sub('\(.*?\)', '', function_name)
+        file_name = match.group(3)
+        return [get_stack_frame(binary, addr, function_name, file_name)]
+      else:
+        return ['%s in %s' % (addr, atos_line)]
+    except Exception:
+      logs.log_error(
+          '%s %s\n%s' % (binary, str(offset), traceback.format_exc(15)))
```

Comment:
please remove traceback.format_exc(15) and its %s from all instances in this file. log_error automatically logs stacktraces, so this is unneeded. this is very old code, so we needed it in old days.

Reply:
Maybe even starting message like Symbolization failed for atos or another place Symbolization failed for llvm-symbolizer for the other similar instance.

Reply:
Done.